test/types/range/elliot/anonymousRangeIter.skipif

```diff
@@ -10,3 +10,6 @@ COMPOPTS <= --no-inline
 
 # test greps generated c code, will fail for llvm
 COMPOPTS <= --llvm
+
+# --no-denormalize creates slightly differnet temporary names
```

Comment:
Are your changes to anonymousRangeIter.expectedGenCode sufficient to handle the slightly differnet temporary names? If they are, skipif should probably remain unchanged.